client/src/components/views/ProfilePage/YourOfferPage/Offers.tsx

```diff
@@ -72,7 +144,11 @@ export const Offers = () => {
               <PriceTypography fontSize="xxs" fontSizeMobile="xs">
                 {offer.price}$
               </PriceTypography>
-              <Icons />
+              <Icons
+                onArchiveButtonClick={onArchiveButtonClickWrapper(offer._id)}
```

Comment:
```suggestion
                onArchiveButtonClick={() => onArchiveButtonClickWrapper(offer._id)}
```
and we wouldnt be needed any strange looking wrappers, the same for two lines below